packages/spacecat-shared-rum-api-client/test/functions.test.js

```diff
@@ -28,4 +29,10 @@ describe('Query functions', () => {
     const notfoundResult = notfound.handler(bundles.rumBundles);
     expect(expected404Result).to.eql(notfoundResult);
   });
+
+  it('crunches experiment data', async () => {
+    const experimentsResult = experiment.handler(bundles.rumBundles);
+    console.log(JSON.stringify(experimentsResult));
+    // expect(expected404Result).to.eql(notfoundResult);
+  });
```

Comment:
I'd rather not commit a test, or clearly mark it skipped, if it's not complete. don't want to give the false impression that this is actually tested down the road if we don't have any expectation yet 😉